### PR TITLE
Goyox86/rustfmting librustc front II

### PR DESCRIPTION
**Pull request by goyox86**
Hi Rustaceans!

This is the second take  on running latest rustfmt on librustc_front!

This is the same in https://github.com/rust-lang/rust/pull/29075 but cleaned. All fixups have been applied.

//cc @nrc

### CODE REVIEW

File: src/librustc_front/lowering.rs [hunk 1]
```diff
@@ -855,8 +856,8 @@ pub fn lower_pat(_lctx: &LoweringContext, p: &Pat) -> P<hir::Pat> {
             }
             PatTup(ref elts) => hir::PatTup(elts.iter().map(|x| lower_pat(_lctx, x)).collect()),
             PatBox(ref inner) => hir::PatBox(lower_pat(_lctx, inner)),
-            PatRegion(ref inner, mutbl) => hir::PatRegion(lower_pat(_lctx, inner),
-                                                          lower_mutability(_lctx, mutbl)),
+            PatRegion(ref inner, mutbl) =>
+                hir::PatRegion(lower_pat(_lctx, inner), lower_mutability(_lctx, mutbl)),
```
Comment (nrc) [inline src/librustc_front/lowering.rs]: this should be braced

Comment (nrc) [inline src/librustc_front/lowering.rs]: https://github.com/nrc/rustfmt/issues/568

**Comment by nrc** [comment]
Thanks for re-doing this - just a couple of things I would fixup (one looks like a bug in rustfmt, but Ok to manually fix it up for now).

**Comment by goyox86** [comment]
Hey @nrc which are the fixups that are needed?

**Comment by nrc** [comment]
@goyox86 Adding braces in lowering.rs, and changing a comment to using `//` in hir.rs (there are comments inline).

**Comment by goyox86** [comment]
Fixups done @nrc

**Comment by nrc** [comment]
Awesome, thanks!

@bors: r+

**Pull request merged**